test/coercion_test.rb

```diff
@@ -10,19 +11,26 @@ def self.call(value)
 
   class Form < TestForm
     feature Coercion
+    include Disposable::Twin::Property::Hash
 
-    property :released_at, type: Types::Form::DateTime
+    property :released_at, type: Types::Params::DateTime
 
     property :hit do
-      property :length, type: Types::Form::Int
-      property :good,   type: Types::Form::Bool
+      property :length, type: Types::Params::Integer
+      property :good,   type: Types::Params::Bool
     end
 
     property :band do
       property :label do
         property :value, type: Irreversible
       end
     end
+
+    property :metadata, field: :hash do
```

Comment:
Is the intention of this addition to test coercion in combo with hash fields? :beers: